packages/@webex/media-helpers/src/webrtc-core.ts

```diff
@@ -32,12 +32,10 @@ export type ServerMuteReason =
   | 'clientRequestFailed' // client called setUserMuted() but server request failed
   | 'localUnmuteRequired'; // server forced the client to be unmuted
 
-// these events are in addition to WCME events. This will be properly typed once webrtc-core event types inheritance is fixed
```

Comment:
I've removed these comments, because they are out-of-date, the types inheritance is already fixed in webrtc-core